chapter09/custom/movielens_download_operator.py

```diff
@@ -0,0 +1,33 @@
+import json
+
+from airflow.models import BaseOperator
+
+from chapter09.custom.movielens_hook import MovielensHook
```

Comment:
There is no pip install of the custom components anywhere. How does this work?

Reply:
Fixed